Arm/rover_arm_system.hpp

```diff
@@ -210,6 +211,19 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   }
 
  private:
+  void VerifyNonZeroes(double & acceleration)
+  {
+    if (acceleration == 0)
+    {
+      acceleration = 0.0001;
+    }
+  }
+  double VectorAddition(double left, double right)
+  {
+    return(left + right);
+  }
```

Comment:
Not sure if this was necessary? Are there going to be edge cases that this will account for in the future?
Because it might actually be cleaner just doing the addition there e.g. 
`accel = accel_x + accel_y;`
as opposed to
`accel = VectorAddition(accel_x, accel_y);`

Reply:
Got it, wasn't sure if that was a good idea or not.